source/_patterns/02-molecules/nav/_nav.twig

```diff
@@ -32,11 +32,13 @@
 ] | sort | join(' ') | trim %}
 
 <{{ nav_element }} class="{{ nav_classes }}" {{ nav_id }} {{ nav_role }}>
-  {% for item in nav_items %}
-    {% if item.nav_item_dropdown %}
-      {% include '@atoms/dropdown/_dropdown.twig' with item %}
-    {% else %}
-      {% include '@molecules/nav/_nav-item.twig' with item %}
-    {% endif %}
-  {% endfor %}
+  {% block nav_items %}
```

Comment:
Verifying that this refactor works back to pl/drupal apps?

Reply:
I spun up a Drupal env using this branch to check. Unfortunately I did it post merge of master. There is something wrong with the menu (the admin menu unsets on the page), but it looks like it's an issue with the underscore library as I'm getting a "Uncaught SyntaxError: Invalid or unexpected token" pointing to underscore in browser console.

I don't think it's the inclusion of the block is a problem for Drupal. Specifically because we had to do a bit of custom work to get this working in Drupal's menus such that we're actually not embedding or including `@molecules/nav/_nav.twig` in the Drupal theme. We write out what we need conditionally, then include the nav-items. I think this is okay, but we should open a ticket for underscore.

tl;dr I think this is still okay to merge.

Reply:
That underscore issue really should be solved here: https://github.com/phase2/particle/issues/259